### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaksymKos**
- [DEMO LINK](https://MaksymKos.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://MaksymKos.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/blocks/card.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 166px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/blocks/card.css:8]: move it to the html selector

File: src/blocks/card.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-count {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
```
Comment (tetiana-ivanchenko) [inline src/blocks/card.css:49]: font-style: normal;
  font-weight: 400; is default values, so they're redundant

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix comments and tests

File: src/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  display: block;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  display: block;
+  margin-bottom: 4px;
+  font-family: inherit;
+  font-style: normal;
```
[the remark this answers is not in the document]
Comment (VitaliyBondarenko1982) [reply src/blocks/card.css:23]: not fixed

File: src/blocks/card.css (hunk 2 above)
Comment (VitaliyBondarenko1982) [reply src/blocks/card.css:51]: not fixed

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by anstsot — APPROVED** [review]
Great job, approved!

File: src/blocks/stars.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
```
Comment (etojeDenys) [inline src/blocks/stars.css:17]:
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-image: url(/images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}

```

File: src/blocks/card.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
```
Comment (etojeDenys) [inline src/blocks/card.css:3]: better to move it to index.css

**Review by etojeDenys — APPROVED** [review]

**Pull request closed without merging**